README.md

```diff
@@ -30,13 +34,25 @@ adding also the necessary headers for cross application tracing.
 The NewRelic agent is currently not set up to support multithreaded requests. As [any of the other client integrations](https://docs.newrelic.com/docs/agents/ruby-agent/features/http-client-tracing-ruby#typhoeus),
 _newrelic-manticore_ will trace all parallel requests as [one external service call](https://github.com/runtastic/newrelic-manticore/blob/master/test/new_relic/manticore/instrumentation_test.rb#L50).
 
-### What about "background"/async requests?
+### What about background or async requests?
```

Comment:
There isn't "this" OR "that" type of request, it's two words for the same thing.

```suggestion
### What about async "background" requests?
```